### PR TITLE
rcon 0.5 (new formula)

### PR DESCRIPTION
**Pull request by Holek**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/rcon.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "pkg-config" => :build
+
+  def install
+    system "cmake", ".", "-DINSTALL_BASH_COMPLETION=OFF", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    assert_equal "No host and/or port specified", pipe_output("#{bin}/rcon").chomp
```
Comment (SMillerDev) [inline Formula/rcon.rb:19]: This takes a second argument as the input. Do you mean `shell_output`?

Comment (Holek) [reply Formula/rcon.rb:19]: yeah, `stderr` to be exact.

Comment (SMillerDev) [reply Formula/rcon.rb:19]: `shell_output` with a ` 2>&1` is the best way

Comment (Holek) [reply Formula/rcon.rb:19]: thanks!

**Comment by Holek** [comment]
okay, I have no idea why my tests are being cancelled

**Comment by SMillerDev** [comment]
I don't know why either but it's been happening all day, it's really annoying. It's not related to your code. Current working theory is a queue issue so maybe wait until the Actions tab queue is empty.

**Comment by SeekingMeaning** [comment]
My workaround right now is waiting for one of the four total "slots" to be open before pushing

https://github.com/Homebrew/homebrew-core/actions?query=is:in_progress

\# of workflows in progress = # of unavailable slots

**Comment by Holek** [comment]
Okay, the only failure in these tests is:

  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)

I wouldn't hold my breath that an RCON tool would get a lot of forks, because it's a [very simple protocol](https://developer.valvesoftware.com/wiki/Source_RCON_Protocol) that has been around with minimal changes since ca. 2007.

I would call this Formula a convenience rather than a necessity. Could we somehow ignore that issue for rcon client and merge it anyway?

**Comment by SMillerDev** [comment]
In that case, while this seems like a nice software. It's not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future, even though you mention it probably won't change.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Comment by Holek** [comment]
Sounds good, I've created a separate tap for that. https://github.com/Holek/homebrew-rcon

I'm adding this comment for anyone who would end up here looking for any rcon utility in Homebrew

**Comment by SMillerDev** [comment]
In that interest, you could also make a PR for https://docs.brew.sh/Interesting-Taps-and-Forks

**Pull request closed without merging**